test/grouping.jl

```diff
@@ -11,9 +11,9 @@ const ≅ = isequal
 d = DataFrame(n = 1:20, x = [3, 3, 3, 3, 1, 1, 1, 2, 1, 1, 2, 1, 1, 2, 2, 2, 3, 1, 1, 2])
 g = groupby(d, :x, sort=true)
 
-@test @based_on(g, nsum = sum(:n)).nsum == [99, 84, 27]
```

Comment:
It would be good to continue testing `@based_on`, e.g. in a separate test/deprecated.jl file.

Reply:
added.